### ISSUE TITLE
Help: I get an error any time I try loading mods into P5R (Steam Deck)

### ISSUE THREAD

**Issue by JoJorre** [Reloaded-Project/Reloaded-II#219]
To start I will say I closely followed the Reloaded II Installation Guide for the Steam Deck [https://gamebanana.com/tuts/15409] and everything appears to be working okay and I have a bunch of mods which successfully installed, however any time I click the P5R icon in Reloaded II where it brings me to my list of installed mods I always get an "Unknown Error" message which I've copied and posted below.

Unknown Error
External component has thrown an exception.
at System.Reflextion.LoaderAllocatorScout.<Destroy>g____Pinvoke|1_0(IntPtr nativeLoaderAllocator)
at System.Reflection.LoaderAllocatorScout.Finalize()

**Comment by Sewer56** [comment]
I have absolutely no idea; whether this might be mod specific; specific to the given wine version, etc.
Sorry.

**Comment by EternisRequiem** [comment]
![16822901015958744627542137527674](https://user-images.githubusercontent.com/20715975/233870302-01fb54cf-94dd-435e-b248-2aa7cc51b457.jpg)

Sorry for grandpa screenshot
This happens to me when I try to drag my mods into a different order.. looks like the first time I happened it saved it but after that I can't do it at all

**Comment by Sewer56** [comment]
Not exactly sure; never seen an error like this before. It might be caused by Wine, might not; extremely strange.

Quick question:

Can you still rearrange mods using any of the other methods?

Here are two:
- Hold Modifier Key (Controller Binding) and press up/down.
or
- Hold CTRL and Press Up/Down if you have a keyboard available.

**Comment by EternisRequiem** [comment]
> Not exactly sure; never seen an error like this before. It might be caused by Wine, might not; extremely strange.
> 
> Quick question:
> 
> Can you still rearrange mods using any of the other methods?
> 
> Here are two:
> 
> * Hold Modifier Key (Controller Binding) [`this is usually X`] and press up/down.
>   or
> * Hold CTRL and Press Up/Down if you have a keyboard available.

Updated to newest version on steam deck (I overwrote files from the release zip - auto update in app crashes and I'm assuming that's fine?)

I tried control + arrow and it crashes but usually I can order things a couple of times but now throws *no* errors and simply closes without warning. Same thing reordering with a click.

More info: 
Rarely when I open reloaded and mouse over the P5R icon it crashes before the floating text box shows up. 
If I right click a mod and I see a slightly dotted artifact show up over the plus symbol, I know it's about to crash.  After right clicking even if it doesn't show up, it tends to crash.

If I just click on a mod, hover over things, etc, does not seem to crash. 


I also moved from Experimental proton to 8.0-1, as I think Experimental updated and that's what led to this stuff if I had to guess? 

(...And also now my console wont load even tho it's checked lol)
